### ISSUE TITLE
[ Dark Mode ] Sometimes the toggle works exactly opposite as it should be

### ISSUE THREAD

**Issue by AnirudhPanda** [smaranjitghose/awesome-portfolio-websites#942]
![reactsnip](https://user-images.githubusercontent.com/66218496/125773042-a6528801-5131-4444-9c80-99f9a0ab28b5.gif)

**Comment by khushi-x** [comment]
The toggle is controlled by Iconify ([on this line](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/travel_temp.html#L36)). Is there anywhere in the project where we modify this?

**Comment by AnirudhPanda** [comment]
> The toggle is controlled by Iconify ([on this line](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/travel_temp.html#L36)). Is there anywhere in the project where we modify this?

Iconify is just used for the icon I guess

**Comment by AnirudhPanda** [comment]
> The toggle is controlled by Iconify ([on this line](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/travel_temp.html#L36)). Is there anywhere in the project where we modify this?

I think this does it, on inspecting I found this
https://github.com/smaranjitghose/awesome-portfolio-websites/blob/5d016cd82a87fa0f7c90113a471cd508326b0a68/assets/js/app.js#L25

**Comment by anushbhatia** [comment]
> > The toggle is controlled by Iconify ([on this line](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/travel_temp.html#L36)). Is there anywhere in the project where we modify this?
> 
> I think this does it, on inspecting I found this
> https://github.com/smaranjitghose/awesome-portfolio-websites/blob/5d016cd82a87fa0f7c90113a471cd508326b0a68/assets/js/app.js#L25

Yes this is the place where toggle button functionality is present.

**Comment by sakshiisaxena** [comment]
Fixed issue [ Dark Mode ] Sometimes the toggle works exactly opposite as it should be.

The error was coming when we used to refresh while being on dark mode. This occurred because the localStorage was not getting cleared every time we refreshed.
Now when you toggle, everything will be perfect. 

https://user-images.githubusercontent.com/68650106/126012108-dc4d71c3-d94b-4ef6-8045-9976c0de3e69.mp4
